azext_iot/tests/utility/test_iot_file_operations.py

```diff
@@ -0,0 +1,59 @@
+from azext_iot.common.fileops import tar_directory, write_content_to_file
```

Comment:
Missing headers, I'm thinking we should add a module linter job to help with this kind of thing